### DISCUSSION TITLE
JDBC with Connection Pooler (pgbouncer)

### DISCUSSION THREAD

**Discussion by shusaan** [cloudnative-pg/cloudnative-pg#3114] [Q&A] [answered]
I have deployed Cluster along with Pooler, Our application is using JDBC, CLoudnative-PG cluster created secret that have jdbc-uri, When I have checked the value the, it's point to primary RW service, like below,
jdbc:postgresql://prod-rw:5432/telemetry?password=zMoUxiSpxpeOdzv&user=telemetry
Now my question is that how can we use JDBC URI with Pooler?

**Comment by gbartolini** [comment, marked as answer]
You should use the service of the pooler. Just run `kubectl get services` and you'll see it.
